apps/order/models.py

```diff
@@ -0,0 +1,74 @@
+import uuid
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+
+class CartItem(models.Model):
+    book = models.ForeignKey(
+        'book.Book',
+        on_delete=models.CASCADE,
+        related_name='book_cart_item',
+        null=True,
+        blank=True,
+        verbose_name=_('Book')
+    )
+    created_by = models.ForeignKey(
+        'user.User',
+        on_delete=models.CASCADE,
+        related_name='cart_item',
+        null=True,
+        blank=True,
+        verbose_name=_('Created by')
+    )
+    quantity = models.PositiveIntegerField(verbose_name=_('Quantity'))
+
+    class Meta:
+        verbose_name = _('Cart Item')
+        verbose_name_plural = _('Cart Items')
+
+    def __str__(self):
+        return f'{self.created_by} - {self.book}'
+
+
+class BookOrder(models.Model):
+    title = models.CharField(max_length=255, verbose_name=_('Title'))
+    price = models.BigIntegerField(verbose_name=_('Price'))
+    quantity = models.PositiveIntegerField(verbose_name=_('Quantity'))
+    isbn = models.CharField(max_length=255, verbose_name=_('ISBN'))
+    edition = models.CharField(max_length=255, verbose_name=_('Edition'))
+    price = models.BigIntegerField(verbose_name=_('Price'))
```

Comment:
Also, reference the book. with SET_NULL

Reply:
Also when we reference to book, do we need to define `title, price` etc here as well, cant we have them from book model?

Reply:
@k9845 We are taking a snapshot for title and price here in case they change or get deleted.